spark/dl/src/test/scala/com/intel/analytics/bigdl/optim/DistriOptimizerSpec.scala

```diff
@@ -157,7 +200,7 @@ class DistriOptimizerSpec extends FlatSpec with Matchers with BeforeAndAfter {
     mm.getParameters()._1.fill(0.125)
     val optimizer = new DistriOptimizer[Double](mm, dataSet, new MSECriterion[Double]())
       .setState(T("learningRate" -> 20.0))
-      .setEndWhen(Trigger.maxEpoch(5))
```

Comment:
5 epochs before was still less iterations (160) than a single epoch now (256). So, as long as the test passes, I don't think this is concerning.